exec/java-exec/src/main/java/org/apache/drill/exec/record/VectorInitializer.java

```diff
@@ -97,7 +97,7 @@ public void allocateBatch(VectorAccessible va, int recordCount) {
     }
   }
 
-  private void allocateVector(ValueVector vector, String prefix, int recordCount) {
+  public void allocateVector(ValueVector vector, String prefix, int recordCount) {
```

Comment:
Note the error in the member variable in this class:

```
   private Map<String, AllocationHint> hints = new HashMap<>();
```

Should be a case insensitive map.